src/structures/BaseGuildVoiceChannel.js

```diff
@@ -37,13 +36,7 @@ class BaseGuildVoiceChannel extends GuildChannel {
    * @readonly
    */
   get members() {
-    const coll = new Collection();
-    for (const state of this.guild.voiceStates.cache.values()) {
-      if (state.channelId === this.id && state.member) {
-        coll.set(state.id, state.member);
-      }
-    }
-    return coll;
+    return this.guild.voiceStates.cache.filter(s => s.channelId === this.id && s.member);
```

Comment:
This is changing the type from `Collection<Snowflake, GuildMember>` to `Collection<Snowflake, VoiceState>`.

Reply:
You are right, my bad - reverting back to the old code as .filter and .map would end up iterating twice over the collection while this is only one iteration.